docs/components/componentOverviewGeneralComponentsList.js

```diff
@@ -238,8 +238,8 @@ const GENERAL: ListItemType = [
   },
   {
     svg: <RadioButton />,
-    name: 'RadioButton',
-    description: 'se RadioButtons when you have a few options that a user can choose from.',
+    name: 'RadioGroup',
```

Comment:
Should we use `Fieldset`'s  svg instead? It's actually more representative. We can remove the RadioButton svg if so